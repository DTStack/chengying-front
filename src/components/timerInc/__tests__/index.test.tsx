import * as React from 'react';
import TimerInc from '../index';
import { RenderResult, cleanup, render } from '@testing-library/react';

describe('timerInc render', () => {
  let wrapper: RenderResult;
  beforeEach(() => {
    wrapper = render(<TimerInc initial={1} />);
  });

  afterEach(cleanup);

  test('timerInc render', () => {
    const time = wrapper.container.getElementsByTagName('div')[0];
    expect(time.textContent).toBe('1s');
  });

  test('timerInc snapshot', () => {
    expect(wrapper.asFragment()).toMatchSnapshot();
  });
});
