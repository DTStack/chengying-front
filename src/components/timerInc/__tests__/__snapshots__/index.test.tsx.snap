// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`timerInc render timerInc snapshot 1`] = `
<DocumentFragment>
  <div>
    1s
  </div>
</DocumentFragment>
`;
